daphne/src/vdaf/mod.rs

```diff
@@ -65,7 +65,7 @@ pub(crate) enum VdafError {
 }
 
 /// A VDAF verification key.
-#[derive(Clone, Deserialize, Serialize)]
+#[derive(Debug, Clone, Deserialize, Serialize)]
```

Comment:
Is `Debug` required for something? We have generally tried to avoid deriving it for cryptographic keys to make it harder for a programmer to accidentally leak its value in a log or something. If we need to implement it, I'd suggest providing one that avoids displaying the value.